DuckDuckGo/BrokenSiteCategories.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 
 struct BrokenSite {
     
-    enum Category: String, CaseIterable {
+    enum Category: String, CaseIterable, Identifiable {
```

Comment:
This is fine, but for future reference if we're only adding `Identifiable` for use in SwiftUI you can do this as a private extension or even just use `id: \.self` in the iterator.